.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,52 @@
+---
+name: Bug report
+about: Report a problem with this library that needs to be fixed.
+title: ''
+labels: triage
+assignees: ''
+
+---
+
+**Before filing this issue**
+Make sure you've carefully reviewed the
+[documentation](https://developers.google.com/google-ads/api/docs/client-libs/python)
+for this library.
+
+This library is only used for the
+[Google Ads API](https://developers.google.com/google-ads/api/docs/start), for
```

Comment:
nit: change comma to period and capitalize For

Reply:
Done